### ISSUE TITLE
GUID does not copy to clipboard on Linux

### ISSUE THREAD

**Issue by rajeev-kumar-sharma** [natewallace/createGUID#2]
1. When i try to create GUID/UUID or random number or short id and click copy, still it do not copy that id on clipboard on Linux
2. vscode 1.5.2

**Comment by natewallace** [comment]
I don't think you are using my extension since it only has a single Create GUID command.  I'm guessing you are using the Create Unique Ids (https://marketplace.visualstudio.com/items?itemName=donjayamanne.createuniqueid) extension which is not mine.

**Comment by rajeev-kumar-sharma** [comment]
I have tried both neither is working

I am using node 6.5.0

**Comment by natewallace** [comment]
Both of our extension are using the [copy-paste](https://www.npmjs.com/package/copy-paste) package to access the clipboard.  The documentation for this package says that it wraps the [xclip](http://www.cyberciti.biz/faq/xclip-linux-insert-files-command-output-intoclipboard/) program on the linux os.  Can you check to see if xclip is present on your machine and if there are any issues with it?

**Comment by rajeev-kumar-sharma** [comment]
Hi,
After installing xclip, extensions are working fine.
Thanks
